packages/android-performance-profiler/src/commands/ScreenRecorder.ts

```diff
@@ -0,0 +1,77 @@
+import { Logger } from "@perf-profiler/logger";
+import { executeAsync } from "./shell";
+import { ChildProcess } from "child_process";
+
+async function waitForFileSizeIncrease(filePath: string): Promise<void> {
```

Comment:
yes because just checking the existence of the file doesn't prove that the video started correctly, maybe the worker created the file then took some time launching the video on the device